planetary-social/nos
# Bio Sheet

## Issues covered
#999 and #1001

## Description

- I added a new view named `BioSheet` that displays the name, nip-05, a mastodon badge (if it is a user coming from mastodon like the one I specified in How to test, and the bio.

- Open that view from ProfileHeaderView when the user taps on the name, nip-05 and bio.

- Got rid of SelectableText. In order to do that, I also had to modify `RawEventView` because not only it used it, but also because it had a few long-lasting bugs: the background was too dark (it was using app-bg), it didn't have a scroll view. So now it looks much better.

## How to test
1. Navigate to a Profile, for example, npub147ra0fhs4aqvxz734dfs2kasryn63fsl9an8yjcyppaqmrry5q8sr5p0ws
2. Tap on the name, nip-05, bio, or read more
3. Interact with the sheet that appears on screen.

## Screenshots/Video

![Simulator Screenshot - iPhone 15 - 2024-05-09 at 17 09 04](https://github.com/planetary-social/nos/assets/1703242/a6a1f027-8032-4e69-9b4d-9f684802db63)
ges.
